web/spiders/spider_ce.py

```diff
@@ -61,5 +57,3 @@ def parse(self, response):
         )
```

Comment:
O ajuste seria aqui. Algo assim:
```python
        if imported_confirmed == imported_deaths == 0:
            imported_confirmed = imported_deaths = None
        self.add_city_case(
            city="Importados/Indefinidos",
            confirmed=imported_confirmed,
            deaths=imported_deaths,
        )
```

Assim garantimos que se não tiver dados pra Importados/Indefinidos, eles entram como `None` e não como `0`.

Reply:
saiba que vc pode enviar essa sugestão de mudança no código, como código mesmo, daí eu poderia simplesmente acatar e fazer a modificação que achar necessário ;)

veja: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request

Reply:
Legal! Não conhecia o `suggestion`. Obrigado :)